sources/packages/backend/apps/db-migrations/src/migrations/1712601354111-UpdateProgramYearDropDownList.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class UpdateProgramYearDropDownList1712601354111
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Update-program-year-drop-down-list.sql", "ProgramYear"),
```

Comment:
Sorry missed to notice this. Please have the file named NOT based on UI perspective. i.e avoid the `drop down` context. 

File name could be `Update-program-year-description.sql`

Reply:
fixed to say description instead of dropdown